ydb/core/kqp/provider/yql_kikimr_exec.cpp

```diff
@@ -1682,9 +1682,24 @@ class TKiSinkCallableExecutionTransformer : public TAsyncCallbackTransformer<TKi
                     for (const auto& indexSetting : listNode) {
                         auto settingName = indexSetting.Name().Value();
                         if (settingName == "indexName") {
-                            auto indexName = indexSetting.Value().Cast<TCoAtom>().StringValue();
-                            auto indexTablePath = NKikimr::NKqp::NSchemeHelpers::CreateIndexTablePath(table.Metadata->Name, indexName);
-                            alterTableRequest.set_path(std::move(indexTablePath));
+                            const auto indexName = indexSetting.Value().Cast<TCoAtom>().StringValue();
+                            const auto indexIter = std::find_if(table.Metadata->Indexes.begin(), table.Metadata->Indexes.end(), [&indexName] (const auto& index) {
+                                return index.Name == indexName;
+                            });
+                            if (!indexIter) {
+                                ctx.AddError(
+                                    TIssue(ctx.GetPosition(indexSetting.Name().Pos()),
+                                            TStringBuilder() << "Unknown index name: " << indexName));
+                                return SyncError();
+                            }
```

Comment:
Where is this tested?

Reply:
Generic error here:
https://github.com/ydb-platform/ydb/pull/7005/files#diff-b80980f1827cf5f57676941f35365f578fb3063b9ef47370835a1af8be6a1dfeR2512

Reply:
I have added new test [AlterTableAlterMissedIndex](https://github.com/ydb-platform/ydb/pull/7005/files#diff-b80980f1827cf5f57676941f35365f578fb3063b9ef47370835a1af8be6a1dfeR2514)